vendors/twtg/models/neon-vs-qt/model.yaml

```
@@ -31,10 +31,17 @@ LoRaWANCertified: false
 # Always mandatory: `<vendorId>:<modelName>:<modelVersion>` (You should have only one per model)
 modelId: twtg:neon-vs-qt:1
 # <vendorId>:<protocolName>:<protocolVersion> Needed for linking the model with a specific driver -> must be the same one used in driver.yaml in the corresponding model (You might have several ones)
-protocolId:
+protocolId: twtg:vs-qt:1
 # DataSheet URL (optional)
 specificationURL: https://github.com/TWTG-R-D-B-V/neon-product-documentation/blob/main/VS-QT/TWTG-NEON_Valve-sensor-QT_Product-Sheet_B1.pdf
```

Although it is correct, I prefer to have the link to the parent folder, such that on document version updates we don't need to update the links everywhere
```suggestion
specificationURL: https://github.com/TWTG-R-D-B-V/neon-product-documentation/tree/main/VS-QT
```

---

Corrected on QT Valve Sensor